### PR TITLE
stricter types

### PR DESCRIPTION
**Pull request by rurban**
esp. multiple definition of `g_content'

### CODE REVIEW

File: src/utils.c [hunk 1]
```diff
@@ -251,7 +252,8 @@ static int _execute_ret(char* msg, int msg_len, const char *cmd) {
   }
 
   if (msg && msg_len > 0) {
-    fread(msg, msg_len - 1, 1, fp);
+      if (!fread(msg, msg_len - 1, 1, fp))
+	  perror("fread");
```
Comment (mwarning) [inline src/utils.c]: Can you fix the indentation here?

File: Makefile [hunk 2]
```diff
@@ -1,38 +1,38 @@
 CC ?= gcc
 CFLAGS ?= -O2
-CFLAGS += -std=gnu99 -Wall -pedantic -Werror -fno-strict-aliasing -Wwrite-strings
-LFLAGS += -lmicrohttpd
-SRC = src/main.c src/webserver.c src/utils.c src/files.c src/call.c
+CFLAGS += -std=gnu99 -Wall -Werror -pedantic -fno-strict-aliasing -Wwrite-strings
+LDFLAGS += -lmicrohttpd
+SRC = src/files.c src/utils.c src/webserver.c src/call.c src/main.c
+NUMWWW := `find www/ -type f | wc -l`
```
Comment (mwarning) [inline Makefile]: This line is not used.

**Comment by mwarning** [comment]
Thank your for the contribution. I thought I was the only one using this project. :)
Please fix the minor issues so I can merge it.

File: Makefile (hunk 2 above)
Comment (rurban) [reply Makefile]: right. I used it as argument for the g_content array, but then went removing it

**Comment by rurban** [comment]
I'm not sure if I will use it, as I'm writing a mesh routing simulators in C also, (not rust).
Based in Atmel's LwMesh, just better (or not). But with different goals than yours.

**Comment by mwarning** [comment]
Anyway, thanks for have a look and have a fun week.

**Comment by mwarning** [comment]
Maybe this might be of interest to you https://github.com/mwarning/meshnet-lab

Atmel's LwMesh is something I have to look into.

**Pull request merged**